src/components/search-result-list-item/search-result-list-item.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./search-result-list-item.styles.scss";
+
+export const SearchResultListItem = ({
+  date,
+  title,
+  readMoretext,
+  readMoreLink,
+  children,
+  color
+}) => {
+  return (
+    <div className={`searchResult ${color}`}>
+      <time className="searchResult__date" dateTime={date}>
+        {date}
+      </time>
+      <h2 className="searchResult__title">{title}</h2>
```

Comment:
Can we decrease a bit the `line-height` for the title?

**Figma:**

![Screenshot 2020-03-28 at 13 15 18](https://user-images.githubusercontent.com/1295800/77822834-8393f900-70f6-11ea-900b-237c324f4270.png)

------

**Implementation:**

![Screenshot 2020-03-28 at 13 16 59](https://user-images.githubusercontent.com/1295800/77822844-9e666d80-70f6-11ea-95cd-86e123dba698.png)